perses/rjmc/topology_proposal.py

```diff
@@ -1137,6 +1138,8 @@ def _construct_atom_map(self,
             key : int (index) , value : str (three letter name of proposed residue)
         new_topology : simtk.openmm.app.Topology
             topology of new system
+        ignore_sidechain_atoms : bool, default False
+            whether to ignore sidechain atoms in the atom map
```

Comment:
this includes CB atoms and beyond?

Reply:
yes. added this clarification to the docstring